### ISSUE TITLE
Fix PeftMixedModel example

### ISSUE THREAD

**Issue by ringohoffman** [huggingface/peft#1824]
### Feature request

The `PeftMixedModel` docstring references `get_peft_model` and imports it in its example but doesn't use it:

https://github.com/huggingface/peft/blob/ad8f7cb59ee7ca4b9ca1c9048711038ac36b31b8/src/peft/mixed_model.py#L97-L107

### Motivation

The unused import in the docstring is confusing.

### Your contribution

Yes.

**Comment by BenjaminBossan** [comment]
Indeed, the import is not needed. Are you interested in creating a PR to fix the docstring?

**Comment by namanvats** [comment]
@BenjaminBossan Created a PR to fix the redundant import: https://github.com/huggingface/peft/pull/1848
